.github/workflows/release.yml

```diff
@@ -729,12 +729,8 @@ jobs:
             fail-fast: false
             matrix:
                 chains:
-                    - chain: stagenet_dancebox
-                      runtime: dancebox
                     - chain: flashbox
                       runtime: flashbox
-                    - chain: dancebox
-                      runtime: dancebox
                     - chain: frontier_template
                       runtime: container-chain-template-frontier
```

Comment:
We may also need to change the url of the frontier_template upgrade job, as its also failing:

[CI / chopsticks-upgrade-test (frontier_template, container-chain-template-frontier)](https://github.com/moondance-labs/tanssi/actions/runs/17493380253/job/49689586803?pr=1242)